charts/hono/values.yaml

```diff
@@ -823,107 +823,247 @@ deviceRegistryExample:
+      # Device Registration service with example device data
+      devicesFile: example-device-identities.json
+      # credentialsFile indicates the name of the file that is used to initialize the
+      # Credentials service with example credentials
+      credentialsFile: example-credentials.json
+      # resetOnStartup indicates whether registration data should be reset to initial
+      # example data when the example Device Registry starts up
+      resetOnStartup: true
+
+    # storageClass indicates the name of the kubernetes StorageClass that
+    # should be used for the example registry's persistent volume claim.
+    # If not set, the cluster's default storage class is used.
+    storageClass:
+
+    # hono contains the Device Registry's configuration properties as defined in
+    # https://www.eclipse.org/hono/docs/admin-guide/device-registry-config/
+    hono:
+      # auth contains Hono client properties used by the example registry for
+      # connecting to the Authentication service.
+      # If not set, the registry by default uses the Auth Server component to
+      # authenticate clients.
+      auth:
+      registry:
+        # amqp contains configuration properties for the server's
+        # exposed AMQP endpoints.
+        # If not set, the registry by default exposes the secure port
+        # using an example key and certificate.
+        amqp:
+        #  insecurePortEnabled: true
+        #  insecurePortBindAddress: "0.0.0.0"
+
+        # rest contains configuration properties for the server's
+        # exposed HTTP endpoints.
+        # If not set, the registry by default exposes the insecure and secure ports
+        # using an example key and certificate.
+        rest:
+        #  insecurePortEnabled: true
+        #  insecurePortBindAddress: "0.0.0.0"
+
+      # healthCheck contains configuration properties for the service's
+      # health check server as defined by
+      # https://www.eclipse.org/hono/docs/admin-guide/monitoring-tracing-config/#health-check-server-configuration
+      # If not set, a TLS secured health check server is configured listening on
+      # all network interfaces on port 8088 using the service's
+      # example key and cert.'
+      healthCheck:
+      #  bindAddress: 0.0.0.0
+      #  port: 12000
+      #  keyPath: "/etc/config/key.pem"
+      #  certPath: "/etc/config/cert.pem"
 
-      # rest contains configuration properties for the server's
-      # exposed HTTP endpoints.
-      # If not set, the registry by default exposes the insecure and secure ports
-      # using an example key and certificate.
-      rest:
-      #  insecurePortEnabled: true
-      #  insecurePortBindAddress: "0.0.0.0"
+  # mongoDBBasedDeviceRegistry contains configuration properties for the
+  # example MongoDB based Device Registry.
+  mongoDBBasedDeviceRegistry:
+    # enabled indicates whether the example MongoDB based Device Registry
+    # should be deployed and used.
+    enabled: false
```

Comment:
this basically means that we can deploy both registries, the file based and the mongo DB based together, right?
how about using a single property *type* with possible values `file` and `mongo` instead?

Reply:
and `jdbc` in the future

Reply:
> and `jdbc` in the future

This PR takes into account that as many types along with that type specific configurations be added in the future. When the _jdbc_ based device registry is there, it can be easily added as a new type along with _jdbc_ specific configuration.